flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/factories/DataGenOptions.java

```diff
@@ -77,6 +78,14 @@
                     .withDescription(
                             "Maximum value to generate for fields of kind 'random'. Maximum value possible for the type of the field.");
 
+    /** Placeholder {@link ConfigOption}. Not used for retrieving values. */
+    public static final ConfigOption<Integer> FIELD_MAX_PAST =
+            ConfigOptions.key(String.format("%s.#.%s", FIELDS, MAX_PAST))
+                    .intType()
```

Comment:
How about using durationType()?
Although it's just placeholder, it'll be more clear if using durationType().